src/main/java/duke/command/InvalidCommand.java

```diff
@@ -0,0 +1,26 @@
+package duke.command;
```

Comment:
Hmm.. interesting that you created an InvalidCommand class to represent invalid user input. If I were you, I would have created a DukeException and catch it in my main Duke class. But interesting approach, and not wrong

Reply:
^ same comment. i think you could create an `InvalidCommandException` that extends from your `DukeException` and make your `Parser` throw that exception.

Reply:
Thanks for the recommendations, i think i will change it to throw a DukeExecption when getting invalid command.

Reply:
Nice.